src/components/PaginationButton.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <button
+    :class="[
+      'pagination-button',
+      $slots.default ? [`pagination-button--${direction}`] : [],
```

Comment:
I think this can be simplified even more
```suggestion
      direction ? `pagination-button--${direction}` : null,
```